[Hub Generated] Review request for Microsoft.KeyVault to add version stable/2019-09-01

heaths opened the pull request:
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/Azure/azure-rest-api-specs/dev-keyvault-Microsoft.KeyVault-2019-09-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

heaths committed:
Added SoftDeleteRetentionInDays, CustomizedRecoverable and recoverableDays (#8457)

Co-authored-by: gdlxm <40441617+gdlxm@users.noreply.github.com>
Co-authored-by: iscai-msft <43154838+iscai-msft@users.noreply.github.com>

AutorestCI commented:
/azp run automation - sdk

majastrz commented on specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json:
```diff
@@ -1122,7 +1122,13 @@
+          "default": true,
+          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. If it's not set to any value(true or false) when creating new key vault, it will be set to true by default. Once it's been set to true value, it can NOT be reverted to false."
+        },
+        "softDeleteRetentionInDays": {
```
>softDeleteRetentionInDays [](start = 9, length = 25)

What's the default value on `softDeleteRetentionInDays` and can we put it in the swagger?

gdlxm replied:
90

majastrz replied:
Thanks for updating

msfcolombo committed:
Added `enableRbacAuthorization` property to Key Vault management plane (#8468)

AutorestCI commented:
/azp run automation - sdk

gdlxm committed:
Add default=90 to softDeleteRetentionInDays (#8476)

* Add default=90 to softDeleteRetentionInDays

Add default=90  to softDeleteRetentionInDays

* Remove default softDeleteRetentionInDays  when patching

Remove default softDeleteRetentionInDays  when patching.

AutorestCI commented:
/azp run automation - sdk

majastrz reviewed (approved):
Signed off from ARM side.

AutorestCI commented:
/azp run automation - sdk

AutorestCI commented:
/azp run automation - sdk

heaths commented:
The validation passed, I checked the two checkboxes in the issue description which retriggered the pipelines, then they failed (i.e. no code changes). @akning-ms can your team look into this?

heaths commented:
/azp run automation - sdk

heaths commented:
/azp run automation - sdk

Timeout in JavaScript SDK generation - the other ones that previous failed without code change passed.

heaths commented:
/azp run automation - sdk
